### DISCUSSION TITLE
Project Collaborators

### DISCUSSION THREAD

**Discussion by christianblandford** [christianblandford/react-daisyui#2] [General]
@benjitrosch Let's use this discussion to plan the future of this project. 

I am happy to try to find a way to combine both of our projects and add you as a collaborator to this one. Looking through your code, it looks somewhat similar to what I wrote, thankfully. It also looks like you have more experience with Storybook and testing than I do (which is very little). It will be amazing to have your assistance with these. I'll also take a closer look at our components and see where the overlap it, and what we could use from both.

**Comment by benjitrosch** [comment]
Sounds good! Maybe we could make some shared repo under the daisyui organization and migrate things over? Our code does look very similar so that'll avoid too much conflict 😄 

I gotta say, love the generators! Such a cool idea to make contributing/creating new components easy.

**Reply by christianblandford** [reply]
This would be ideal, though I don't think that org currently exists. If @saadeghi would be open to it, I would love to go this route and not have the project under my personal org.

**Reply by saadeghi** [reply]
@christianblandford @benjitrosch That's great.  
There is https://github.com/daisyui I added you both.  
You should be able to add new repos. Let me know If I can help 💙

**Reply by benjitrosch** [reply]
Thanks @saadeghi, amazing! Just made an empty private react-daisyui repo

**Reply by christianblandford** [reply]
Thanks @benjitrosch  and @saadeghi! some exciting stuff happening here

**Comment by benjitrosch** [comment]
@christianblandford Just added you as a collaborator on my repo as well!

**Comment by christianblandford** [comment]
I wish I knew more about how plop works, because those generators are really hacked together... But for sure they are powerful. Would be nice to be able to configure common props like size, color/variant, etc... automatically when generating. I'll have to do some more research there.

**Comment by christianblandford** [comment]
@benjitrosch can you add me as a collaborator to the new repo and open up discussions so we can move this conversation over there

**Reply by benjitrosch** [reply]
Done! Just added you and made the discussion section as well 🙂
